homeassistant/components/media_player/plex.py

```diff
@@ -410,8 +415,21 @@ def refresh(self, device, session):
             if self._session.index is not None:
                 self._media_episode = str(self._session.index).zfill(2)
 
-        # Music
-        if self._media_content_type == MEDIA_TYPE_MUSIC:
+        if self._session_type == 'clip':
```

Comment:
Let's just combine these and drop the debug output. We don't need to continuously print that while a clip is playing. Then we can avoid creating `_tvshow_info()`.
```python
if self._session_type in ['clip', 'episode']:
```

Reply:
I like that, yes that's a much better way do to it. Its actually how i wanted to but didnt know how..